### PR TITLE
Con 2755 application overview page

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

File: src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml [hunk 1]
```diff
@@ -71,6 +95,17 @@
...
                         </div>
                     }
+                    @if (Model.IsWithdrawal)
+                    {
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">Contact details</dt>
+                            <dd class="govuk-summary-list__value">
+                                @Model.ContactName
+                                <br />
+                                <a href="mailto:@Model.ContactEmail" class="govuk-link">@Model.ContactEmail</a>
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml:105]: There may be a thing about no using mailto - I've had this in the past query with deen or design team?

File: src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml [hunk 2]
```diff
@@ -36,27 +36,51 @@
...
                         </div>
                     }
-                    @if (!string.IsNullOrWhiteSpace(Model.LegalName))
+                    @if (!string.IsNullOrWhiteSpace(Model.Standard) && Model.SequenceNo == ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO)
                     {
                         <div class="govuk-summary-list__row">
-                            <dt class="govuk-summary-list__key">Registered name</dt>
-                            <dd class="govuk-summary-list__value">@Model.LegalName</dd>
+                            <dt class="govuk-summary-list__key">Details</dt>
+                            <dd class="govuk-summary-list__value">@Model.Standard</dd>
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml:57]: It doesn't look like this includes the STRef - I think it should from the AC

File: src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml [hunk 3]
```diff
@@ -36,27 +36,51 @@
                             <dd class="govuk-summary-list__value">@Model.ApplicationReference</dd>
                         </div>
                     }
+                    @if (!string.IsNullOrWhiteSpace(Model.LegalName) && Model.IsWithdrawal)
+                    {
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">Organisation name</dt>
+                            <dd class="govuk-summary-list__value">@Model.LegalName</dd>
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml:43]: This needs checking for correct name against the use on the tab view organisation name

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml (hunk 1 above)
Comment (Howesy77) [reply src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml:105]: Not sure - this is how it is in the design

Comment (Howesy77) [reply src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml]: Removed mailto:

File: src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml (hunk 2 above)
Comment (Howesy77) [reply src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml]: Changed to ST Ref

File: src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml (hunk 3 above)
Comment (Howesy77) [reply src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml]: Changed to EPOA Assessor Name

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]
There should also be similar change to the Sequence_ReadOnly.cshtml - maybe there are missing AC's - or Deen just assumed you would know

**Comment by Howesy77** [comment]
> There should also be similar change to the Sequence_ReadOnly.cshtml - maybe there are missing AC's - or Deen just assumed you would know

Done

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**